src/components/Account/Account.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ArrowIcon from "../../assets/icons/flechita.svg";
+import ShoppingCartIcon from "../../assets/icons/icon_shopping_cart.svg"
+import './accounNavBar.scss';
+
+const Account = () => {
+    return (
+        <div className="NavBarAccount">
```

Comment:
estas siguiendo algún sistema para las clases? BEM, por ejemplo?

Reply:
I really have a problem using BEM . I don't know how use it. But I was thinking in search how other people use BEM in Yard Sales.